fidap/settings.py

```diff
@@ -1,5 +1,2 @@
 import os
-from dotenv import load_dotenv, find_dotenv
```

Comment:
Please rollback these changes, this code is for local development if you add .env file to root directory with BASE_URL=localhost:8000 then it will target localhost